backend/src/charityprojects/models.py

```diff
@@ -68,4 +68,17 @@ class LearnNewSkill(models.Model):
     newSkill = models.CharField(max_length=255)
     description = models.TextField(blank=True, null=True)
     video = models.FileField(upload_to='upload/challengeVideo', null=True)
-    pu_id = models.ForeignKey(ProjectUser, on_delete=models.CASCADE, null=True)
\ No newline at end of file
+    pu_id = models.ForeignKey(ProjectUser, on_delete=models.CASCADE, null=True)
+
+
+class UserInvitation(models.Model):
+    objects = None
+    pu_id = models.ForeignKey(ProjectUser, on_delete=models.CASCADE, null=True)
+    user_id = models.ForeignKey(User, on_delete=models.CASCADE)
```

Comment:
'user_id' not needed because we can get 'user_id' from 'pu_id'. Also update the usage in the method 'update_user_invitation' if you remove it from here